frontend/app/package.json

```diff
@@ -18,7 +18,7 @@
     "electron:package": "electron-builder build --config .electron-builder.config.js --publish onTag",
     "electron:serve": "node scripts/serve.js",
     "i18n:report": "vue-i18n-extract report --vueFiles './src/**/*.?(ts|js|vue)' --languageFiles './src/locales/**/*.json'",
-    "link:verify": "ts-node --esm scripts/check-external-links.ts",
+    "link:verify": "ts-node scripts/check-external-links.ts",
```

Comment:
for some reason esm mode works locally but not on CI, but we also don't have the project yet as "type": "module"